woocommerce/woocommerce-admin
# Rename Google Shopping image asset.

Attempt to avoid overly-aggressive ad blockers.

### Detailed test instructions:

- Enable an ad blocker in your browser
- Go through the new OBW
- Verify the Google Shopping icon is not blocked
